pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools>=45",
+    "setuptools_scm[toml]>=6.0",
+    "ninja",
+    "cmake>=3.18.2",
```

Comment:
Mind that `manif` support goes back to Ubuntu 18.04 which [ships with CMake 3.10.2](https://packages.ubuntu.com/search?keywords=cmake).

Reply:
Yes, this is correct. However, this is the CMake version that is installed in the virtualenv created automatically during the wheel build process. I used this version on purpose since it's [the recommendation from pybind11](https://pybind11.readthedocs.io/en/stable/compiling.html#findpython-mode). Such CMake versions include a much better Python logic, and since this does not affect in any case the user's setup, I used a more recent require version. In most cases, at the time on writing, the build process will pull CMake >= 3.20.